test/integration/features/basic.feature

```diff
@@ -113,6 +121,13 @@ Feature: Basic test
         Then stdout should contain "To login as a regular user, run 'oc login -u developer -p developer"
         And stdout should contain "To login as an admin, run 'oc login -u kubeadmin -p "
 
+    @darwin @linux @windows
+    Scenario: Monitoring stack check
+        Given checking that CRC is running
+        When login to the oc cluster succeeds
+        And executing "oc get pods -n openshift-monitoring" succeeds
```

Comment:
It doesn't come up as soon as the we exclude it from CVO override, it does take some time so may be you need to put some retry here.

Reply:
Updated the description of the PR to reflect changes brought in by #1787. That also addresses your concern -- `crc status` is executed before we check the pods, and status waits for the monitoring operator. Should be OK now.